api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/EditN3GeneratorVTwo.java

```diff
@@ -78,8 +79,14 @@ public void subInMultiUris(Map<String,List<String>> varsToVals, List<String> n3t
         }         
     }
   
-    private boolean containsNullOrEmpty(List<String> values) {        
-        return values != null && ( values.contains(null) || values.contains("") );       
+    private boolean containsNullOrEmpty(List<String> values) {
+    	//return values != null && ( values.contains(null) || values.contains("") );
```

Comment:
This may be a bit knitpicky but we can probably remove this line before merging.